app/models/repository_issue/bitbucket.rb

```diff
@@ -40,7 +40,7 @@ def self.create_from_hash(name_with_owner, issue_hash, token = nil)
 
       i.repository_id = repository.id
       i.labels = [issue_hash.fetch(:metadata, {})[:kind]]
-      i.pull_request = issue_hash[:type] == 'pull_request'
+      i.pull_request = issue_hash[:type] == 'pullrequest'
```

Comment:
Is this a bug that's existed for a whiel?

Reply:
It's something I noticed while working through this. I'm not sure when that value changed, but it was what I was seeing in the live API responses. We also have a mix of V1 and V2 API calls to Bitbucket and it could differ between the two?